ground/src/test/java/com/google/android/ground/repository/LocationOfInterestRepositoryTest.kt

```diff
@@ -93,11 +94,9 @@ class LocationOfInterestRepositoryTest : BaseHiltTest() {
   fun testApplyAndEnqueue_enqueuesLoiMutation() = runWithTestDispatcher {
     locationOfInterestRepository.applyAndEnqueue(mutation)
 
-    locationOfInterestRepository
-      .getIncompleteLoiMutationsOnceAndStream(LOCATION_OF_INTEREST.id)
```

Comment:
The semantics here are a bit difference than getSurveyMutations -- if this was the only use of this method, I think that's fine, just calling it out in case.

Reply:
I wanted to deprecate `getIncompleteLoiMutationsOnceAndStream` without removing this test case. So, used `getSurveyMutations` as a fallback.